move/utils/sources/utils/utils.move

```diff
@@ -0,0 +1,52 @@
+/// Module: utils
+module utils::utils;
+
+use sui::bcs::{Self, BCS};
+
+// -----
+// Macros
+// -----
+
+/**
+* Peel data from a BCS encoded vector
+* @dev This macro is used to peel data from a BCS encoded vector
```

Comment:
Don't think natspec format should be used. Use the same as other move code / Rust
https://github.com/MystenLabs/sui/blob/db5ba7aa9564af91de0aa9ad58ac0f2c9f86eac2/crates/sui-framework/packages/sui-framework/sources/kiosk/kiosk_extension.move#L62